### DISCUSSION TITLE
'make clean' not cleaning up intermediate files set by 'objdir'?

### DISCUSSION THREAD

**Discussion by somyungoh** [premake/premake-core#1768] [Q&A] [answered]
Hello,

I am facing an issue with `gmake`. I have intermediate files configured by using `objdir`. However, `make clean` does not cleanup these intermediate files but only the final binaries. 

I am wondering if this is normal, and if that's the case, is there a way I can bind custom deletion to the `make clean` command?

Thanks,

**Comment by starkos** [comment, marked as answer]
Not normal. Consider [opening an issue](https://github.com/premake/premake-core/issues) and providing the information requested so we can try to help out.

**Reply by somyungoh** [reply]
Okay thanks. I'll continue the discussion in the Issues.
